testing/benchmarking/main.tf

```diff
@@ -33,6 +33,10 @@ provider "aws" {
   region = var.aws_region
```

Comment:
All the AWS resources will be tagged if you specify a `default_tags` field: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags

Reply:
Do you mean instead of passing the tags to the tf modules ? I don't think we can assume they are using aws